### DISCUSSION TITLE
bigfix/10.0/compliance/Compliance/bf_client_compliance

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#37] [Q&A]
# bigfix/10.0/compliance/Compliance/bf_client_compliance

This guide describes the BigFix Client Compliance and its solutions (BigFix Client Compliance Configuration and BigFix Client Compliance Windows (IPSec Framework)) which helps in maintaining the compliance status of the computers.

https://help.hcltechsw.com/bigfix/10.0/compliance/Compliance/bf_client_compliance.html

**Comment by omomsweet** [comment]
test1
